### DISCUSSION TITLE
航空氣象的重要性

### DISCUSSION THREAD

**Discussion by youtinglin** [1101-GS4719-C/Python-Community-GS4719-C#1530] [General]
要放暑假啦~
大家有計畫出國玩嗎?
相信因為疫情的減緩且流感化，讓已經多年沒出國的人已經迫不及待了。

> ### 航空氣象是什麼?

出國肯定會搭飛機的，那麼你知道在每個機場都會有航空氣象台，會有航空氣象人員在那裡堅守著?
要有一趟安全的飛行旅程，除了必須仰賴機師的操控技術以外，還需要依靠航空氣象人員蒐集、整理、分析和解釋飛航所需的機場及航路之預測、預報、警報及顯著危害天氣資訊。

各民航機場都有設置的自動氣象觀測系統（Automatic Weather Observation System，簡稱 AWOS），方便航空氣象人員進行機場天氣測報和航機管制作業。而有時會在機場附近看到非常大顆的圓球，那就是雷達。
在桃園國際機場的就是都卜勒氣象雷達，是C 波段氣象雷達，掃描範圍可達 300 公里。能夠掌握劇烈天氣如颱風、雷雨、風切和亂流等天氣現象的發展和移動，提供資訊給飛航相關人員作業參考，確保飛行安全。

不知道大家在出國的同時有沒有想過一群人默默為大家的安全堅守岡位嗎?

**Comment by lin157644** [comment]
強風大雨都會飛機的起降帶來挑戰，並對乘客造成威脅，雖然我自己搭乘飛機的經歷並不多，但也很感謝這些航空氣象人員守護我們出遊安全，也謝謝你的分享讓我得知他們的努力。

**Comment by poyen0806** [comment]
沒錯，所以可以使用 Python 的套件來進行天氣資訊的爬取與預測，例如：
1. Requests 或 BeautifulSoup爬取天氣
2. pandas處理資訊
3. matplotlib 和 seaborn視覺化資訊
4. Scikit-learn 或 TensorFlow/Keras預測
5. PyOWM快速測試預測結果
